content/docs/Configuration/_index.md

```diff
@@ -21,13 +21,67 @@ Both Packit Service and Packit CLI use this configuration file.
 Here you can see a list of options that can be defined at the top-level and
 shared by all of the jobs.
 
+#### packages
+
+(*dict*) This key was introduced mainly to enable support for monorepositories, upstream repositories
+mapping to multiple downstream packages. It holds a dictionary of `{<package_name>: <package_object>}`.
```

Comment:
Not sure if `<package_object>` will tell anything to people. Maybe mentioning the package properties that used to be specified top level?

The example makes it clear, but still.

Also, would be nice to define what options are available.